src/components/Books.js

```diff
@@ -1,19 +1,44 @@
 /* eslint-disable react/prop-types */
```

Comment:
Let's remove all disabled Linters, Linting helps in cleaner and professional code ✊ 

Kindly check for other occurrences 🙂